packages/components/psammead-media-player/src/Amp/index.test.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { render } from '@testing-library/react';
 import Amp from '.';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Usually want library imports above of our own files - surprised eslint isn't whinging about this, guess the rule is only in Simorgh